examples/cephfs/storageclass.yaml

```diff
@@ -41,6 +41,11 @@ parameters:
   # If omitted, default volume mounter will be used - this is
   # determined by probing for ceph-fuse and mount.ceph
   # mounter: kernel
+
+  # (optional) Prefix to use for naming volumes.
```

Comment:
nit: I guess "optional" is not required explicitly when in the next line it is mentioned that if omitted it would default to some value. So IMO, its self explanatory.
Same for other files.

Reply:
> nit: I guess "optional" is not required explicitly when in the next line it is mentioned that if omitted it would default to some value. So IMO, its self explanatory.
> Same for other files.

Thanks. Since it was a convention for this file, changed everywhere else but here.